libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/sync/SyncState.kt

```diff
@@ -14,3 +14,11 @@ enum class SyncState {
     Terminated,
     Offline,
 }
+
+fun SyncState.isConnected() = when (this) {
```

Comment:
I know the name was from me initially, but maybe we should instead call it `isOffline`?
`fun SyncState.isOffline() = this is SyncState.Offline`.

Or even an extension method on the `SyncService`
`fun SyncService.isOffline(): StateFlow<Boolean> = syncState.mapState { it == SyncState.Offline }`

Reply:
Can we do the opposite
`fun SyncService.isOnline(): StateFlow<Boolean> = syncState.mapState { it != SyncState.Offline }`
it will be simpler to use.